packages/nodes-base/nodes/Stellar/actions/entities/IPriceFraction.ts

```diff
@@ -0,0 +1,4 @@
+export interface IFraction {
+	n: string;
+	d: string;
```

Comment:
Same here.

Reply:
These mean numerator and denominator. I will make them explicit.